src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -279,11 +279,13 @@ public void Dispose()
 
             private void SendCommands()
             {
-                //fire and forget, do not hold-up smuggler operations waiting for Raft command
-                AsyncHelpers.RunSync(() => _database.ServerStore.SendToLeaderAsync(new AddOrUpdateCompareExchangeBatchCommand
+                AsyncHelpers.RunSync(async () =>
                 {
-                    Commands = _compareExchangeCommands
-                }));
+                    using (_database.ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+                    {
+                        return await _database.ServerStore.SendToLeaderAsync(new AddOrUpdateCompareExchangeBatchCommand(_compareExchangeCommands, context));
+                    }
```

Comment:
I had some doubts with this change because the comment suggested we are fire and forget here while we actually did it in sync manner so we always wait for the operation to be applied.

Reply:
The comment is wrong

Reply:
I think this is a copy/paste issue